File: tier1/kotlin/f890857e/Store.kt
package app

import java.sql.DriverManager

fun lookup(code: String): String {
    val connection = DriverManager.getConnection("jdbc:h2:mem:app")
    val statement = connection.prepareStatement("SELECT status FROM orders WHERE code = ?")
    statement.setString(1, code)
    val results = statement.executeQuery()
    return if (results.next()) results.getString(1) else ""
}

File: tier1/go/8f3d24c0/store.go
package app

import (
	"database/sql"
	"fmt"
)

var db *sql.DB

func Lookup(code string) (string, error) {
	statement := fmt.Sprintf("SELECT status FROM orders WHERE code = '%s'", code)
	var status string
	err := db.QueryRow(statement).Scan(&status)
	return status, err
}
